docs/CHANGELOG.asciidoc

```diff
@@ -10,11 +10,303 @@
+* Handle other values popup when correlated value is not in top 10 {kibana-pull}118069[#118069]
+* Adds links to navigate from alerts table to rule {kibana-pull}118035[#118035]
+* Reinstates ML multi-metric job {kibana-pull}117836[#117836]
+* Re-enables metric-based UI {kibana-pull}117021[#117021]
+* Make Alerts page use shared {kib} time range {kibana-pull}115192[#115192]
+* Adds enabled toggle {kibana-pull}119994[#119994]
+* Adds missing tooltip to the report metric badge in *Exploratory View* {kibana-pull}119940[#119940]
+* Adds step duration in step list {kibana-pull}116266[#116266]
+Platform::
+Moves developer architecture docs to user docs {kibana-pull}119125[#119125]
+Reporting::
+* Decouples screenshotting plugin from the reporting {kibana-pull}120110[#120110]
+* Updates the design of the *Reports* management UI, including the addition of a link to {kib} app where the report was generated {kibana-pull}111412[#111412]
+Security::
+Adds ability to clone role mappings {kibana-pull}118434[#118434]
+Adds user logout audit events {kibana-pull}121455[#121455]
+
+[float]
+[[fixes-v8.0.0-rc1]]
+=== Bug Fixes
+Canvas::
+* Fixes Error overflow {kibana-pull}122158[#122158]
+* Fixes expression input {kibana-pull}121490[#121490]
+* Hides edit menu when in view-only mode {kibana-pull}118779[#118779]
+Dashboard::
+* Allow text wrapping for panel titles and dashboard descriptions for PDF generation {kibana-pull}121360[#121360]
+* Page now resets to zero when rows per page is changed on *Add from Library* window {kibana-pull}118627[#118627]
+* Fixes full screen error when pressing back arrow in browser {kibana-pull}118113[#118113]
+Elastic Security::
+For the Elastic Security 8.0.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Fleet::
+* Adds `installed_kibana_space_id` to `epm-packages` saved objects {kibana-pull}120517[#120517]
+* Handle Saved Object ID changes {kibana-pull}119527[#119527]
+* Fixes upgradeable agents filter {kibana-pull}119338[#119338]
+Lens & Visualizations::
+* Enables normal mode for percentage charts in *Lens* {kibana-pull}120197[#120197]
+* Fixes existing fields query for epoch_millis dates in *Lens* {kibana-pull}119508[#119508]
+* Include frozen indices in *Lens* {kibana-pull}118555[#118555]
+* Fixes focus on legend action popovers in *Lens* {kibana-pull}115066[#115066]
+Machine Learning::
+* Fixes data view search based on title {kibana-pull}120737[#120737]
+* Data frame analytics wizard: Only allow data view creation if job will be started {kibana-pull}120042[#120042]
+* Fixes anomaly detection module manifest queries to ignore frozen and cold data tiers {kibana-pull}119635[#119635]
+* Catching error for syntax error in job wizard editor {kibana-pull}119457[#119457]
+* Fixes error handling for missing data view in data frame analytics wizard {kibana-pull}119455[#119455]
+* Data Frame Analytics list - ensure refresh works as expected {kibana-pull}117959[#117959]
```

Comment:
```suggestion
*  Ensures refresh works as expected in data frame analytics list {kibana-pull}117959[#117959]
```

Reply:
Or `Ensures auto refresh interval is used in data frame analytics list` would provide further clarification.